wireapp/kalium
# fix: obfuscate sensitive info in network logs

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

after obfuscate the log messages using regex it affects the performance so we needed to find another way


### Solutions

convert the messages in the ktor response and request to JSon so we can detect the sesiteve info

#### How to Test

run using this branch and see if you still see any sensitive info in the network level that not obfuscated

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
